CRM/Financial/BAO/FinancialAccount.php

```diff
@@ -361,12 +361,15 @@ public static function validateFinancialAccount($financialAccountId, $financialA
    * with Financial Account
    *
    * @param array $params
+   *   Holds submitted formvalues and params from api for updating/adding contribution
    *
    * @param int $contributionID
+   *   Contribution ID
    *
    * @param obj $form
+   *   Holds form object of CRM_Contribute_Form_Contribution
    *
-   * @return string
+   * @return bool
    *
    */
   public static function checkFinancialTypeHasDeferred($params, $contributionID = NULL, $form = NULL) {
```

Comment:
Its used later part in code to generate line items When creating new
contribution from backoffice.